db/schema.rb

```diff
@@ -20,8 +20,8 @@
     t.integer  "user_id"
     t.integer  "recipe_id"
     t.text     "action"
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
+    t.datetime "created_at"
```

Comment:
I'm not sure why your install keeps trying to change the schema.rb file in this way, but those columns should stay null:false.